keras_cv/layers/preprocessing/random_cutout.py

```diff
@@ -162,17 +160,14 @@ def _parse_bounds(self, factor):
         else:
             return type(factor)(0), factor
 
-    def call(self, inputs, training=True):
-        if training is None:
-            training = backend.learning_phase()
+    def get_random_transformation(self):
```

Comment:
I think this is probably not correct. This function should generate the transformation for one image, and in future bbox and label with it. So it should not only contain the number of images, but also the position/size of the cutout. When the augment_image got the transformation, it can augment the image without any random behavior involved.

Reply:
it seems that this method might need the actual input (for the size of the image) to produce all the transformations. We probably will need to update the method signature in base class. @LukeWood

Reply:
Ah I see - this is an interesting issue.  I guess we could `vectorized_map(images, get_random_transformation)`?

Reply:
Sending https://github.com/keras-team/keras/pull/16299.

Reply:
https://github.com/keras-team/keras/pull/16299 is now merged and thus this issue can be solved.  @quantumalaviya

Reply:
you will need to upgrade to the latest keras-nightly.

Reply:
I'll work on this. I think I will wait for #217 to be completely merged before making the changes.

Reply:
it is as merged as it will be for awhile - thanks.  Just need to upgrade keras nightly!

Reply:
I've updated this - we should remove num_cutouts for now.  It doesn't work as we want